examples/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "testbed_viewer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "serve-example": "webpack serve --config webpack.dev.js",
```

Comment:
because it's such a standard convention, should we have a `start` script in there too?

Reply:
you have to run the build process in the root directory first before you can serve this. My thinking was that a developer would always be running things from the root directory to avoid confusion of moving up and down directories